examples/gallery/embellishments/timestamp.py

```diff
@@ -15,7 +15,7 @@
 import pygmt
 
```

Comment:
Kimda unrelated, but at L13, did we want to use the first or second figure for the thumbnail in the gallery?

Reply:
Sounds a good idea to use the second one as the thumbnail.

Reply:
From my side, using the second figure as the thumbnail is fine. I changed this in commit https://github.com/GenericMappingTools/pygmt/pull/2418/commits/77a0ae3a23e8a617d1cdbe3e2c7f336938822124.
Hope @michaelgrund also agrees, since he added this example 🙂.